lib/lang.d.ts

```diff
@@ -4,8 +4,8 @@ import {
 
 export function isUndefined(obj: any): obj is null | undefined;
 export function isDefined(obj: any): obj is Exclude<any, null | undefined>;
-export function isNil(obj: any): value is null | undefined;
-export function isArray(obj: any): obj is Array;
+export function isNil(obj: any): obj is null | undefined;
+export function isArray(obj: any): obj is Array<unknown>;
```

Comment:
Will put this back to any match what it was prior to the commit noted in this PR's description
```suggestion
export function isArray(obj: any): obj is Array<any>;
```